### ISSUE TITLE
Newer Tensorflow builds available? And Python3.

### ISSUE THREAD

**Issue by wadetb** [ROCmSoftwarePlatform/hiptensorflow#36]
Hi, for my project I've been using `tf-nightly-gpu` which offers some new features, the one that is important to me is the `contrib_audio` module for its spectrogram and mfcc operators. For now I have switched to using `librosa` but I would like to go back to using Tensorflow for this if possible.

Are there plans to release versions of hipTensorflow built against more recent versions of Tensorflow?

Additionally, my project was developed with Python 3 and it's cost me some time eliminating the use of features such as `pathlib`. It would be nice to have hipTensorflow available for Python 3.

Thanks!

**Comment by sunway513** [comment]
We are working on PR to upstream the HIP backend to TensorFlow, that'll be the path we catch up with more recent TF releases. 
Have you found any issues of using Python 3 for hipTF?

**Comment by wadetb** [comment]
Thanks for letting me know your plans with respect to upstream. 

Regarding Python3, the provided 1.0.1 Tensorflow build is built for Python 2.7 and is incompatible with Python 3.x. For example, if you run the hipTF docker image and type `python3` and try to `import tensorflow`, you will see an error.

```
root@149b3d2351d4:/srv/ira/ira/data# python3
Python 3.5.2 (default, Sep 14 2017, 22:51:06)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'tensorflow'
```

For my purposes I have discovered that `pathlib` is available for Python 2.x via `pip install pathlib`, and I've been able to separate out my MFCC code into another module (which I run using `tf-nightly` on the CPU), and any remaining missing Tensorflow APIs have been worked around.
